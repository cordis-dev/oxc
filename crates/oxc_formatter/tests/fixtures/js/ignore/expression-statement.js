// prettier-ignore
Object.defineProperties    (    exports    , {
})

/* prettier-ignore */
(() => 
  c +
    b +
  d
)