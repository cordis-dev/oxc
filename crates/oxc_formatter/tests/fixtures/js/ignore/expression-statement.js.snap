---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
// prettier-ignore
Object.defineProperties    (    exports    , {
})

/* prettier-ignore */
(() => 
  c +
    b +
  d
)
==================== Output ====================
// prettier-ignore
Object.defineProperties    (    exports    , {
})

/* prettier-ignore */
(() => 
  c +
    b +
  d
)

===================== End =====================
